docs/changes/stats/14533.feature.rst

```diff
@@ -0,0 +1,3 @@
+This pull request is to address the lack of "weights" and "axis" parameters on the circstats function ``vonmisesmle``.
```

Comment:
I would rephrase the change log text to the tone of what you want user to know about the changes when they read the release change log. They would not care it was from a pull request. Feel free to look at existing change log texts for examples. Thanks!